modules/quick-starts-highlighting-reference.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="quick-start-highlighting-reference_{context}"]
+= Quick start highlighting markdown reference
+
+The highlighting, or hint, feature enables Quick Starts to contain a link that can highlight and animate a component of the web console.
+
+The markdown syntax contains:
+
+* bracketed link text
+* the `highlight` keyword, followed by
+* the ID of the element that you want to animate
```

Comment:
I wouldn't trail between second and third bullets here, but would instead just combine them into one bullet:
"The `highlight` keyword, followed by the ID of the element that you want to animate"